packages/py/polywrap-core/tests/test_get_implementations_spec.py

```diff
@@ -57,4 +57,5 @@ def test_interface_implementations_not_redirected():
 
     get_implementations_results = get_implementations(Uri(interface1_uri), interfaces, redirects)
 
+    ## comment: is this assertion correct using the `in` operand? the Jest version is a `.toEqual` instead.
```

Comment:
I think this is a good catch and indeed we should be using `==` instead of in 🤔

Reply:
updated to use `==` instead 👍